pennylane_forest/qvm.py

```diff
@@ -110,53 +123,19 @@ def __init__(self, device, *, shots=1024, noisy=False, **kwargs):
         elif isinstance(device, str):
             self.qc = get_qc(device, as_qvm=True, noisy=noisy, connection=self.connection)
 
+        if timeout:
```

Comment:
The default value of `timeout` is `None`, so this should probably be
```suggestion
        if timeout is not None:
```

Reply:
But this is quite nice to now be able to set!